### DISCUSSION TITLE
Schema Field reporting_entity_name Clarification

### DISCUSSION THREAD

**Discussion by ghost** [CMSgov/price-transparency-guide#36] [Q&A] [answered]
The field reporting_entity_name found in all 3 files is defined as, "The legal **name of the entity publishin**g the machine-readable file."

Does publishing imply the entity name that is **publishing** the machine-readable file on their website or the entity name that is **creating** the file that could be on a website of the one creating the file or for another entity to publish on their website?

For example a insurance carrier, ABC Health,  
- creates a in-network-rates file and publishes it on the public website, ABCHealth.com. The reporting_entity_name would contain "ABC Health";
- ABC Health has their pharmacy claims processed by an external pbms vendor, Rx Claims R Us, and the pbms vendor creates the prescription drug file for ABC Health to publish on their website, ABCHealth.com. Would the reporting_entity_name contain "ABC Health" or "Rx Claims R Us"?;
- ABC Health is providing a self funded group, ACME Manufacturing, a in-network-rates file for the self funded group to publish on their website ACMEManufact.com. Would the reporting_entity_name contain "ABC Health" or "ACME Manufacturing"?

**Comment by shaselton-usds** [comment, marked as answer]
Thanks for the question. 

The intent for the `reporting_entity_name` element was to identify the entity that was actually making the file. Not the entity that was hosting the file (which would most likely be the issuer/payer).

In your second example, the question of "ABC Health" or "Rx Claims R Us", the `reporting_entity_name` would be "Rx Claims R Us".

In your third example, the question of "ACME Manufacturing" or "ABC Health", the `reporting_entity_name` would be "ABC Health".

**Reply by ghost** [reply]
Thank you!
